modules/microshift-deploying-a-load-balancer.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-deploying-a-load-balancer_{context}"]
+= Deploying a TCP load balancer on a workload
+
+{product-title} offers a built-in implementation of network load balancers. This process uses the node IP address as the external IP address for the `LoadBalancer` service configuration file. 
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`)
+* Access to the cluster as a user with the cluster-admin role.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
+* The KUBECONFIG environment variable is set.
+
+.Procedure 
+
+. Verify that your pods are running by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -A
+----
+
+. Create a namespace with the following commands. 
++ 
+[source,terminal]
+----
+$ NAMESPACE=nginx-lb-test
+----
++
+[source,terminal]
+----
+$ oc create ns $NAMESPACE
+----
++
+This deploys a test application that verifies the connection.
+
+. Deploy three replicas of the test `nginx` application in your namespace.
++
+[source,terminal]
+----
+$ oc apply -n $NAMESPACE -f https://raw.githubusercontent.com/openshift/microshift/main/docs/config/nginx-IP-header.yaml
```

Comment:
We should not really point to GitHub URLs in the docs like this. They are prone to change and some users will have security concerns about downloading content and applying it directly. We can show the contents of that file in the docs instead (like we do below on line 59), but before we do that we will want to mirror the image being used so we can be certain that it is a safe image to tell customers to use.

Reply:
Yeah I think it would also be better to show the content directly.